card-app/src/UserCard.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import getUsers from './getUsers';
+
+const useStyles = makeStyles({
+  root: {
+    maxHeight: 1000,
+    maxWidth: 545,
+  },
+});
+
+const UserCard = () => {
+  const [user, setUser] = React.useState({
+    avatar:'loading...',
+    email:'loading...',
+    first_name: 'loading...',
+    last_name:'loading...'
+  });
+
+  const [isLoading, setIsLoading] = React.useState(true);
+
+  React.useEffect(() => {
+    getUsers('3')
```

Comment:
Nice work building out the `getUsers` function. One small note is that since we're only fetching one user, I'd probably name the function `getUser` instead - just a small change, but it helps let other developers know what to expect.

Reply:
One other thing here - right now, this component is only capable of rendering out the user with the ID of `'3'`, but it could be capable of doing way more. Instead of hard-coding this value here, I'd have the component take in a prop called `userId` that you could use to fetch the data. That way, you could render out this component a bunch of different times.